### PR TITLE
refactor: move EmptyScreen/FilteredEmptyScreen to ui package

### LINKED ISSUE #6919: Implement EmptyScreen/FilteredEmptyScreen UI component

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #6919

### What does this PR do?

Move EmptyScreen/FilteredEmptyScreen to ui library

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#6919 

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/ui/src/lib/screen/EmptyScreen.svelte [hunk 1]
```diff
@@ -56,7 +58,7 @@ let copyTextDivElement: HTMLDivElement;
         <div class="font-mono text-gray-400" bind:this="{copyTextDivElement}" data-testid="copyTextDivElement">
           {commandline}
         </div>
-        <button title="Copy To Clipboard" class="ml-5" on:click="{() => copyRunInstructionToClipboard()}"
+        <button title="Copy To Clipboard" class="ml-5" on:click="{copyRunInstructionToClipboard}"
```
Comment (axel7083) [inline packages/ui/src/lib/screen/EmptyScreen.svelte]: Could we make it use the `Button` component ?

**Review by axel7083 — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]
As per @axel7083 we should use Button now and needs to be rebased on other changes, but otherwise LGTM.

File: packages/ui/src/lib/screen/EmptyScreen.svelte [hunk 2]
```diff
@@ -21,10 +24,11 @@ onMount(() => {
   processed = true;
 });
 
-function copyRunInstructionToClipboard() {
+function copyRunInstructionToClipboard(): void {
   const text = copyTextDivElement?.textContent;
   if (text) {
-    window.clipboardWriteText(text);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (window as any).clipboardWriteText(text);
```
Comment (benoitf) [inline packages/ui/src/lib/screen/EmptyScreen.svelte]: I think this part is problematic as it's a custom API of Podman Desktop exposed to window object, but it won't work if you're like in a webview

Comment (jeffmaury) [reply packages/ui/src/lib/screen/EmptyScreen.svelte]: Should be solved now

File: packages/ui/src/lib/screen/EmptyScreen.svelte [hunk 3]
```diff
@@ -14,17 +17,19 @@ export let hidden = false;
...
   }
   processed = true;
 });
 
-function copyRunInstructionToClipboard() {
+function copyRunInstructionToClipboard(): void {
   const text = copyTextDivElement?.textContent;
   if (text) {
-    window.clipboardWriteText(text);
+    dispatch('click');
```
Comment (benoitf) [inline packages/ui/src/lib/screen/EmptyScreen.svelte]: I would not name the function copyToClipboard as it's the caller that decide what to do (could be clipboard copy but could be something different)

Comment (benoitf) [reply packages/ui/src/lib/screen/EmptyScreen.svelte]: also, shouldn't it dispatch the text of the element ?

either we need to remove the text variable or we should dispatch the value

**Review by benoitf — APPROVED** [review]

**Pull request merged**